massgov/openmass
# DP-22536:  Make paragraph for Locations on org page

**Description:**
Converted Organization Page Map fields into an Organization Locations paragraph.

**Jira:** (Skip unless you are MA staff)
https://massgov.atlassian.net/browse/DP-22536


**To Test:**
- Login and go to /node/add/org_page (or edit an existing org_page).
- Click on the "Content" tab.
- Expand the "Section Content" field.
- [ ] Verify there is an "Add Organization Locations" button.
- Click the "Add Organization Locations" button.
- [ ] Verify these fields are present: Map, Location details button label, Location button short description, and Nearby.
- Add content to the fields and save (recommend editing an existing org_page with Map content).
- [ ] View the org_page and verify the content displays as expected.

**Screenshots/GIFs:**
![screencapture-mass-local-orgs-massachusetts-registry-of-motor-vehicles-2021-07-20-16_22_19](https://user-images.githubusercontent.com/1434979/126390573-4ec03158-01ed-4e6f-a0d6-f0c1071c444b.png)

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
